jenkins-jobs/gen3-run-load-tests.groovy

```diff
@@ -118,6 +126,21 @@ pipeline {
 
                           SELECTED_LOAD_TEST_DESCRIPTOR=""
 
+                          if [ "$TARGET_ENVIRONMENT" == "qa-dcp" ]; then
+                            echo "b/c target env is qa-dcp, using qa-dcp mLTS client cert and tokens"
+                            mv "$QA_DCP_MTLS_CERT" "\$JENKINS_PATH/mtls.crt"
```

Comment:
IIRC jenkins has ${WORKSPACE} set to the workspace directory out-of-the-box ... just fyi